content/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.md

```diff
@@ -28,8 +29,26 @@ function test() {
 **Tip:** To preserve your formatting within a list, make sure to indent non-fenced code blocks by eight spaces.
 
 {% endtip %}
+{% note %}
+
+**Note:** To display triple backticks in a fenced code block, wrap them inside quadruple backticks.
+
+<!-- {% endnote %} -->
```

Comment:
I think you can omit the note formatting (this is on me for not making my comment clearer when you asked for advice 😄) (Deleting the extra `endnote` tag should also fix the failing test):

```suggestion

To display triple backticks in a fenced code block, wrap them inside quadruple backticks.

```